examples/kubernetes/cluster/main.tf

```diff
@@ -13,12 +9,12 @@ resource "vkcs_kubernetes_cluster" "k8s-cluster" {
   }
 
   name                = "k8s-cluster"
-  cluster_template_id = data.vkcs_kubernetes_clustertemplate.ct.id
-  master_flavor       = data.vkcs_compute_flavor.k8s.id
+  cluster_template_id = data.vkcs_kubernetes_clustertemplate.example_template_by_version.id
```

Comment:
DO NOT USE IN OBJECT NAME SUCH WORDS LIKE examle, resource, datasource, object type name

Reply:
understood